extension_templates/annotation.py

```diff
@@ -30,6 +30,9 @@
     fitted state flag       - is_fitted (property)
     fitted state inspection - check_is_fitted()
 
+Testing
+    get default parameters for test instance - get_test_params()
```

Comment:
`create_test_instance` should also be mentioned, even if extenders don't have to implement it